src/main/java/com/tabnine/general/StaticConfig.java

```diff
@@ -22,11 +23,20 @@
 import java.util.*;
 import javax.swing.*;
 import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
 
 public class StaticConfig {
   // Must be identical to what is written under <id>com.tabnine.TabNine</id> in plugin.xml !!!
   public static final String TABNINE_PLUGIN_ID_RAW =
       Config.IS_ON_PREM ? "com.tabnine.TabNine-Enterprise" : "com.tabnine.TabNine";
+  public static final String TABNINE_ENTERPRISE_URL_PROPERTIES_KEY =
+      TABNINE_PLUGIN_ID_RAW + ".enterpriseUrl";
```

Comment:
is this an entry in the settings?

Reply:
yes: https://github.com/tabnine/tabnine-jb-self-hosted-updater/blob/df93ee214a7d17e5fdb04d5bd779cf11ee002c9f/src/main/kotlin/com/tabnine/Consts.kt#L7